### PR TITLE
Add mason to "chapel master" Dockerfiles

### PR DESCRIPTION
**Pull request by awallace-cray**
These updates were developed in preparation for Chapel release 1.18.0

- Updates debian version to 9.5.
- Builds Chapel mason
- Apt-install a git client and configures a dummy git user, so that mason unit tests work.
- Sets default Linux locale to LANG=en_US.UTF-8

### CODE REVIEW

**Comment by awallace-cray** [comment]
@tjstavenger-pnnl (cc @ben-albrecht): 
GitHub's Web UI did not let me ask you to review this Tim, but I am sure you could comment. 
I've just added "mason" to the Dockerfiles. 
I plan to use these Dockerfiles to build the upcoming Chapel release 1.18.0 docker images. 
Thanks.

File: util/dockerfiles/master/Dockerfile [hunk 1]
```diff
@@ -28,12 +29,17 @@ RUN mkdir -p /opt/chapel \
...
+    && make -C $CHPL_HOME mason
 
 # Configure locale
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     echo 'LANG="en_US.UTF-8"'>/etc/default/locale && \
     dpkg-reconfigure --frontend=noninteractive locales && \
     update-locale LANG=en_US.UTF-8
 
+# Configure dummy git user
+RUN git config --global user.email "noreply@example.com" && \
```
Comment (ben-albrecht) [inline util/dockerfiles/master/Dockerfile]: Is this necessary?

**Comment by tjstavenger-pnnl** [comment]
Adding mason should be fine for what I was trying to do. Do you need the git config for the mason build? I hadn't needed it to download the tarball and build Chapel itself.

**Comment by awallace-cray** [comment]
I wrote a comment somewhere, but it got lost- there's a mason unit test ("registry") and it throws an error when Git cannot find any configured user. It seemed easiest to just add one to the Docker image.

**Comment by awallace-cray** [comment]
I would like to try to agree on the default-locale question from this PR 
https://github.com/chapel-lang/chapel/pull/11026
before merging this PR.  Hopefully we can avoid multiple flavors.
If #11026 is resolved the way I'd like, then these `master\Dockerfiles` would be redundant.

**Pull request merged**